apps/fluent-tester/src/TestComponents/Menu/MenuScrollView.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { ButtonV1 as Button } from '@fluentui/react-native';
+import { Menu, MenuItem, MenuTrigger, MenuPopover, MenuList } from '@fluentui-react-native/menu';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { TextInput, StyleSheet, View } from 'react-native';
+import { commonTestStyles as commonStyles } from '../Common/styles';
+
+const styles = StyleSheet.create({
+  menuContainerRow: { flexDirection: 'row' },
+  menuMargin: { marginRight: 20 },
+});
+
+export const MenuScrollView: React.FunctionComponent = () => {
+  const [maxHeight, setMaxHeight] = useState<number>(800);
+  const ScrollViewMenuPopover = MenuPopover.customize({ maxHeight: maxHeight });
```

Comment:
Defining this inside a function component means it is recreated on every render. Furthermore, the name feels misleading to me. `ScrollViewMenuPopover` sounds like a special menu popover with a scrollview. But really, the scrollview is in the menu list, and this is just setting the max height. Could we just set the Max height via a style prop on line 59?